listenbrainz_spark/popularity/listens.py

```diff
@@ -91,11 +94,13 @@ def get_table_prefix(self) -> str:
     def get_base_path(self) -> str:
         return LISTENBRAINZ_POPULARITY_DIRECTORY
 
-    def get_filter_aggregate_query(self, existing_aggregate: str, incremental_aggregate: str) -> str:
+    def get_filter_aggregate_query_coarse(self, existing_aggregate: str, incremental_aggregate: str,
```

Comment:
this function has the same name as the one above. Intended?

Reply:
There are two classes in this file, so once in each.